### PR TITLE
Several examples refurbished

### PR DESCRIPTION
**Pull request by casaper**

### CODE REVIEW

**Comment by casaper** [comment]
@macrozone ready for review. Examples with default initial state content for better display of the features

File: src/SimpleReadonly.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import '@react-page/plugins-background/lib/index.css';
+import { simpleReadonlyDefaultState } from './SimpleReadonly-default-state';
+
+const plugins = {
+  content: [slate()],
+  layout: [background({ defaultPlugin: slate() })],
+};
+
+export const SimpleReadonlyExample = () => {
+  const [editorValue] = useState(simpleReadonlyDefaultState);
```
Comment (macrozone) [inline src/SimpleReadonly.tsx]: remove useState here

File: package.json [hunk 2]
```diff
@@ -7,20 +7,31 @@
...
-    "@react-page/editor": "^0.26.0",
-    "@react-page/plugins-background": "^0.26.0",
-    "@react-page/plugins-slate": "^0.26.0",
-    "@react-page/react-page": "^0.26.0",
+    "@react-page/editor": "^0.27.0",
+    "@react-page/plugins-background": "^0.27.0",
+    "@react-page/plugins-image": "^0.27.0",
+    "@react-page/plugins-slate": "^0.27.0",
+    "@react-page/plugins-spacer": "^0.27.0",
+    "@react-page/react-page": "^0.27.0",
```
Comment (macrozone) [inline package.json]: we can remove the meta package @react-page/react-page"

Comment (casaper) [reply package.json]: done

File: src/SimpleReadonly.tsx (hunk 1 above)
Comment (casaper) [reply src/SimpleReadonly.tsx]: done

**Pull request merged**